### PR TITLE
Detekt TooManyFunctions rule relaxation

### PR DESCRIPTION
**Pull request by kidinov**
Relaxation for the TooManyFunctions rule

```
TooManyFunctions:
    ignoreOverridden: true
```

**Merge only after the target branch is merged**

### CODE REVIEW

File: config/detekt/baseline.xml [hunk 1]
```diff
@@ -69,16 +69,13 @@
...
     <ID>ForbiddenComment:ProductDetailFragment.kt$ProductDetailFragment$// TODO: add tracking events here</ID>
     <ID>ForbiddenComment:ProductDetailFragment.kt$ProductDetailFragment$// TODO: remove the null checks once the root cause is identified is fixed</ID>
     <ID>ForbiddenComment:ProductDetailRepository.kt$ProductDetailRepository$// TODO: add event to track sku availability success</ID>
     <ID>ForbiddenComment:RefundByItemsFragment.kt$RefundByItemsFragment$// TODO: Temporarily disabled, this will be used in a future release - do not remove</ID>
     <ID>ForbiddenComment:ShippingCarrierRatesViewModel.kt$ShippingCarrierRatesViewModel$// TODO: Once we start supporting countries other than the US, we'll need to verify what currency the shipping labels purchases use</ID>
     <ID>ForbiddenComment:MainActivity.kt$MainActivity$// TODO: Extract logic out of MainActivity to reduce size and remove this @Suppress("LargeClass")</ID>
     <ID>FunctionOnlyReturningConstant:Screen.kt$Screen.Companion$fun isVisible(): Boolean</ID>
-    <ID>LargeClass:OrderDetailViewModelTest.kt$OrderDetailViewModelTest : BaseUnitTest</ID>
```
Comment (hichamboushaba) [inline config/detekt/baseline.xml:81]: Just a question, what did trigger those unrelated changes to the baseline?

Comment (kidinov) [reply config/detekt/baseline.xml:81]: I regenerated baseline and this violation has gone because the class is not that large anymore =)

Comment (hichamboushaba) [reply config/detekt/baseline.xml:81]: That's good to know, I didn't think we refactored this test class.
Thanks for the clarification

**Review by hichamboushaba — APPROVED** [review]
LGTM, thanks for working on this @kidinov
